static/include/components/servo-control.md

```diff
@@ -0,0 +1,7 @@
+## Test the servo
+
+After setting up and establishing the connection for your servo, navigate to the [Control tab](https://docs.viam.com/manage/fleet/robots/#control) and select the dropdown panel dedicated to the motor.
+
+![The servo component in the control tab](assets/components/servo/servo-control-tab.png)
```

Comment:
```suggestion
![The servo component in the control tab](/components/servo/servo-control-tab.png)
```

Very close - the image will work if you remove the assets prefix. The way hugo includes pictures is by looking into the assets folder - so you only provide the file path from there